.github/workflows/ci-publish.yml

```diff
@@ -84,6 +87,5 @@ jobs:
           TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
As we are using poetry rather than twine, this injection of PyPI credentials needs to be updated